ssd-tf2/network.py

```diff
@@ -0,0 +1,165 @@
+from tensorflow.keras import Model 
```

Comment:
What does this for?

Reply:
This is an old file that should be deleted. I'll delete this in the next commit.